From 0a06bbe0440c45b6e935f6bdfc4f7c4f9d7293d2 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 30 May 2016 16:13:57 +0300
Subject: [PATCH 503/538] dts: 8040: enable KR on GOP0 and RGMII on GOP3 for
 CP1 and CP0

-CP0: GOP0 - KR, GOP2 - disabled, GOP3 - RGMII
-CP1: GOP0 - KR, GOP2 - disabled, GOP3 - RGMII
-8040A0 board support on GOP0 KR mode and RGMII on GOP3
-GOP2 doesn't supported on 8040
-default config 2 - 10 Gbit ports and 2 - 1 Gbit ports

Change-Id: I8eb277b81928e44b4ab8c04a4035de2a666dfb84
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30162
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-router.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
index 94b2f01..5c140e7 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
@@ -84,7 +84,7 @@
 				emac2: mac2 {
 					phy-mode = "rgmii"; /* lane-0 */
 					phy = <&phy2>;
-					status = "okay";
+					status = "disabled";
 				};
 				emac3: mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
@@ -97,7 +97,7 @@
 					status = "okay";
 				};
 				eth1: eth1@020000 {
-					status = "okay";
+					status = "disabled";
 				};
 				eth2: eth2@030000 {
 					status = "okay";
@@ -130,7 +130,7 @@
 				emac3_1: mac3 {
 					phy-mode = "rgmii"; /* rgmii-1 */
 					phy = <&phy3>;
-					status = "disabled";
+					status = "okay";
 				};
 			};
 			ppv22@000000 {
@@ -141,7 +141,7 @@
 					status = "disabled";
 				};
 				eth2_1: eth2@030000 {
-					status = "disabled";
+					status = "okay";
 				};
 			};
 		};
-- 
1.9.1

